po/vi.po

```diff
@@ -0,0 +1,362 @@
+# Vietnamese translation by Vietnam Linux L10n <https://github.com/linux-l10n-vi>
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the pantheon-tweaks package.
+msgid ""
+msgstr ""
+"Project-Id-Version: pantheon-tweaks\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-10-26 11:19+0900\n"
+"PO-Revision-Date: 2025-12-31 02:56+0700\n"
+"Last-Translator: Loc Huynh <https://github.com/hthienloc>\n"
+"Language-Team: \n"
+"Language: vi\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.8\n"
+
+#: data/pantheon-tweaks.desktop.in:3 data/pantheon-tweaks.metainfo.xml.in.in:5 src/MainWindow.vala:20
+msgid "Pantheon Tweaks"
+msgstr "Pantheon Tweaks"
+
+#: data/pantheon-tweaks.desktop.in:4
+msgid "Customization Tool"
+msgstr "Công cụ tùy chỉnh"
+
+#: data/pantheon-tweaks.desktop.in:5 data/pantheon-tweaks.metainfo.xml.in.in:6
+msgid "Tweak Pantheon settings"
+msgstr "Cài đặt Tweak Pantheon"
```

Comment:
`Tweak` is not a proper noun but just a verb and capitalized because of the top of the sentence here.